projects/managed-hubs/support.md

```diff
@@ -1,118 +1,217 @@
-  This channel is public, so can be useful if we wish to invite conversation from non 2i2c team members.
+Guidance Request
+  A Support Request that is not tied directly to a change in infrastructure. Sometimes support requests are not tied to a specific change, but a desire to discuss and request guidance. In this case we may set up a meeting to discuss as a group, or have some back-and-forth via support channels.
+```
 
-## Lifecycle of a support request
+### Other support request terms
 
+The following are other important terms and ideas in the support process.
 
-Below is a summary of the major steps that make up the support process, from beginning to end of a ticket.
+```{glossary}
+Support Budget
+  A fixed amount of time that we can spend providing support for each community that we serve. This helps us ensure that we can sustainably serve many communities. Any support request that is **not tied to an {term}`Incident`** will draw from the support budget for that community. If a community requests support beyond their support budget, we may request extra funds to help cover our costs.
 
-:::{note}
-We should try to respond to all support-related communications within one working day.
-:::
+  :::{note}
+  We currently keep this term intentionally vague, and ask that communities are respectful of our time when making change requests.
+  We are investigating the support budget that we should give to each community, and will update here when we have specific numbers in mind. 
+  :::
 
-- When you receive a ticket, communicate that you have received it and note the next steps you’ll follow to resolve it (even if this just means investigating because you're not sure what is wrong).
-- If needed, open a [support issue and add it to the backlog](support:issues).
-  Note that we should still use Freshdesk for support communication on this request, the GitHub issue is just for 2i2c team members to coordinate.
-- Identify an engineering team member that can pick up this work.
-- Give the user updates every day or two as we work to resolve the issue.
-- When it is resolved, tell the user what you’ve done to resolve the issue.
-- Close any FreshDesk tickets and support issues that are related to this request.
-- In some cases, it might be useful to also write down [a private note in the FreshDesk ticket](https://support.freshdesk.com/support/solutions/articles/37580-private-notes-for-internal-sharing)
-  with a short summary of the situation and/or add links to relevant discussions
-  from other platforms.
+Community Representative
+  A person that has the authority to speak on behalf of the communities we work with, and is a point-of-contact for support.
+  See [](roles:community-representative) for more information.
 
-## How to prioritize support requests
+Post Mortem
```

Comment:
IMO 'incident report' is a better term and we should use that consistently rather than post mortem

Reply:
whoops - yep agree